Lib/test/test_io.py

```diff
@@ -591,15 +591,23 @@ def test_large_file_ops(self):
         with self.open(support.TESTFN, "w+b") as f:
             self.large_file_ops(f)
 
+    def _checked_open(self, path, mode, bufsize, under_check=False):
+        if not under_check and bufsize == 1:
```

Comment:
What is `under_check` for? It doesn't seem to be ever True.

Reply:
This function was a conditional context manager, and `under_check` is set to `True` in the recursive call. But I've removed that function anyway to have only one place with warning check (in test_file.py).